templates/experiments/components/experiment_chat.html

```diff
@@ -30,13 +32,15 @@
                 </div>
               </td>
               <td class="px-3 py-4 text-sm text-gray-500">{{ message.content|render_markdown }}</td>
-              {% flag "chat_annotations" %}
-                <td scope="col" class="w-1/4 overflow-y-visible">
-                  <div class="my-2 mr-2">
-                    {% include "generic/tag_multiselect.html" with object_tags=message.get_linked_tags object_info=message.object_info %}
-                  </div>
-                </td>
-              {% endflag %}
+              {% if taggible|default:False %}
```

Comment:
This is to prevent showing the tag component when the user ends the experiment chat and is taken to the transcript view

Reply:
Could this be replaced with a permission check? If the user is logged in a has permissions then it seems OK to show the UI at that stage in the workflow.